runtime/testnet/src/config/assets.rs

```diff
@@ -29,6 +29,8 @@ parameter_types! {
 
 parameter_types! {
 	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
+	// Key = 68 bytes (4+16+32+16), Value = 44 bytes (4+32+8)
+	pub const NftsBalanceDeposit: Balance = deposit(1, 112);
```

Comment:
As above.

Reply:
Even I changed the test to return the direct max size value from the storage info ([here](https://github.com/r0gue-io/pop-node/pull/413#discussion_r1894768254)), it still return `112` which is different from the benchmarking. Wonder if I am missing something, are there any extra overhead when running the actual code?

Reply:
The tests was incorrect because it used the configuration of mock runtime in the pallet not devnet / testnet / mainnet. Resolved by moving the test to its specific runtimes and updated the comments.